Documentation/JSON schema guide.md

```diff
@@ -31,6 +31,16 @@ A `Rule` can be composed by a simple rule type (`String`) or a logical operator
 | `OR` | OR operation on an array with at least two rule types. | `{"OR":[{"NOT": "is_gold_member"}, "is_male", "has_devices"]}` |
 | `NOT` | NOT operation on a rule type. | `{"NOT":"is_gold_member"}` |
 
+### General meta schema
+
+Meta fields that apply to all `Component`s.
+
+| Key | Type | Description |  Optional | Default value |
+| --- | ---- | ----------- |  -------- | ------------- |
+| `id` | `String` | A string that (uniquely) identifies this `Component`. | Yes | `nil` |
+
+Note: `id` is a general purpose discriminator (e.g. distinguish between several instances of the same type). It's up to you if you use this and what you use it for.
+
```

Comment:
The "Component schema" section describes a different level; the `id` would go inside the `meta` dictionary (one level deeper). 

That's why I created a new meta schema section, analogous to the TabBarConfig or StackConfig meta schema sections, if that makes sense? 

And I think I'll rename that from "General meta schema" to "Common meta schema", sounds a bit nicer.